src/tree_data.h

```diff
@@ -258,6 +258,17 @@ struct lyd_node_leaf_list {
 #define LY_VALUE_USER 0x02    /**< flag for a user type stored value */
 /* 0x80 is reserved for internal use */
 
+/**
+ * @brief Anydata value union
+ */
+typedef union {
+    const char *str;             /**< string value, in case of printing as XML, characters like '<' or '&' are escaped */
+    char *mem;                   /**< raw memory (used for LYB format) */
+    struct lyxml_elem *xml;      /**< xml tree */
+    struct lyd_node *tree;       /**< libyang data tree, does not change the root's parent, so it is not possible
+                                      to get from the data tree into the anydata/anyxml */
+} LYD_ANYDATA_VALUE;
```

Comment:
Upper case names are so far used only for macros and enums, so please name the union in lower case

Reply:
done